<launch>
<arg name="model" default='$(find urdf_robot)/urdf/dynamic_robot.urdf' />
<arg name="gui" default="False" />
<param name="robot_description" textfile="$(arg model)" />
<param name="use_gui" value="$(arg gui)"/>
<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
<rosparam param="source_list">["calculated_trajectory"]</rosparam>
<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
<node name="InterpolTraject" pkg="urdf_robot" type="InterpolTraject" />
<node name="FkSolverKDL" pkg="urdf_robot" type="FkSolverKDL" />
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find urdf_robot)/rviz/my_robot.rviz" required="true" />
</launch>
